omod/src/main/webapp/resources/js/view/operation.js

```diff
@@ -176,14 +188,14 @@ define(
                 
             },
 
-            updateStatus: function(status) {
+            updateStatus: function(status, cancelReason) {
```

Comment:
Instead of adding the cancelReason as a parameter, why not just grab it if the status is Cancelled?